### DISCUSSION TITLE
/respawn

### DISCUSSION THREAD

**Discussion by Slyrex** [winstxnhdw/lc-hax#378] [Ideas]
What happened to the /respawn command? I saw a commit it got merged into /heal; but heal says player is dead.

**Comment by winstxnhdw** [comment]
It's just `/heal`. You probably typed `/heal something`. `/heal` alone will not print anything.

**Reply by joep26020** [reply]
sometimes it will respawn you but the spectate hud is still there and you cant interact with anything, but only us m3 + e. no picking up things either. basically its just being in spectate mode but able to walk around.

**Reply by winstxnhdw** [reply]
Isn’t that how `respawn` always works? Is it any different in Project Apparatus?

**Reply by joep26020** [reply]
some times i get it to respawn with no issues like that? so im confused.

**Reply by winstxnhdw** [reply]
The next time you get the issue, send me the logs.

**Reply by joep26020** [reply]
[lc-hax.log](https://github.com/winstxnhdw/lc-hax/files/14235320/lc-hax.log)
just got the issue where i could not respawn

**Reply by winstxnhdw** [reply]
I don’t see an error. Hmm. It’s still showing the previous error you’re getting. Can you delete this logfile and try replicating it again.

**Reply by joep26020** [reply]
yep will do. ill give you two. one where i cant respawn at all and one where it just shows the hud but i am in game

**Reply by joep26020** [reply]
might be a while until tmrw tho

**Reply by joep26020** [reply]
so i deleted the log file and now its not coming back

**Reply by winstxnhdw** [reply]
Means there’s no error.
